test/general/buffer_wrapper.pass.cpp

```diff
@@ -48,13 +48,13 @@ main()
     std::size_t size = 1000;
     sycl::buffer<uint32_t> buf{size};
     test_buffer_wrapper test{};
-    auto begin = sycl::host_accessor<uint32_t, 1, sycl::access_mode::read>(buf).get_pointer();
+    auto data_ptr = sycl::host_accessor<uint32_t, 1, sycl::access_mode::read>(buf).get_pointer();
 
-    test(oneapi::dpl::begin(buf), oneapi::dpl::end(buf), begin, size);
-    test(oneapi::dpl::begin(buf, sycl::write_only), oneapi::dpl::end(buf, sycl::write_only), begin, size);
-    test(oneapi::dpl::begin(buf, sycl::write_only, sycl::noinit), oneapi::dpl::end(buf, sycl::write_only, sycl::noinit), begin, size);
-    test(oneapi::dpl::begin(buf, sycl::noinit), oneapi::dpl::end(buf, sycl::noinit), begin, size);
+    test(oneapi::dpl::begin(buf), oneapi::dpl::end(buf), data_ptr, size);
+    test(oneapi::dpl::begin(buf, sycl::write_only), oneapi::dpl::end(buf, sycl::write_only), data_ptr, size);
+    test(oneapi::dpl::begin(buf, sycl::write_only, __noinit), oneapi::dpl::end(buf, sycl::write_only, __noinit), data_ptr, size);
+    test(oneapi::dpl::begin(buf, __noinit), oneapi::dpl::end(buf, __noinit), data_ptr, size);

```

Comment:
I'd suggest to test both: `sycl::no_init` and `sycl::property::noinit`. We need to verify that both options work

Reply:
Added.